autopilot/betweenness_centrality.go

```diff
@@ -206,10 +206,30 @@ func (bc *BetweennessCentrality) Refresh(graph ChannelGraph) error {
 		}
 	}
 
-	// Divide by two as the graph is undirected.
+	// Get min/max to be able to normalize
```

Comment:
should this be squashed into the commit where the metric is added?

Reply:
left it separate to make it easier to review, now squashed.